src/javascript/binary/websocket_pages/user/account/settings/authorised_apps/authorised_apps.js

```diff
@@ -35,8 +35,12 @@ var ApplicationsUI = (function(){
         var $viewButton = $viewButtonSpan.children(".button").first();
         $viewButton.text(text.localize("Revoke access"));
         $viewButton.on("click",function(){
-            ApplicationsData.revokeApplication(data.app_id);
-            $row.css({ opacity: 0.5 });
+            if(window.confirm(
+                text.localize('Are you sure that you want to permanently revoke access of application') +
```

Comment:
change to "Are you sure that you want to permanently revoke the applications's access?"